cli/Cargo.toml

```diff
@@ -1,25 +1,26 @@
 [package]
-name = "yatima_cli"
+name = "yatima-cli"
 description = "A dependently typed programming language for the decentralized web"
 version = "0.1.0"
-authors = ["John C. Burnham <john@yatima.io>", "Anders C. Sørby <anders@yatima.io>", "Gabriel Barreto <gabriel@yatima.io>", "Samuel Burnham <samuel@yatima.io>"]
+authors = ["John C. Burnham <john@yatima.io>", "Anders C. Sørby <anders@yatima.io>", "Gabriel Barreto <gabriel@yatima.io>", "Samuel Burnham <sam@yatima.io>"]
 edition = "2018"
 
 [[bin]]
 name = "yatima"
 path = "src/main.rs"
 
 [dependencies]
-yatima_core = { path = "../core"}
-yatima_utils = { path = "../utils"}
-cid = "0.6.1"
-multihash = "0.13.2"
+yatima-core = { path = "../core"}
+yatima-utils = { path = "../utils"}
+cid = { git = "https://github.com/yatima-inc/sp-ipld" }
```

Comment:
Why are we importing these individually, shouldn't we be able to import one `sp_ipld` crate that has cid, multihash and ipld as subcrates?